tests/test-remove-ad-server.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+
+source "${SCRIPT_DIR}/common.sh"
+
+if [ ${KEEP} -eq 1 ]; then
+	echo "keeping ad server deployment (KEEP=1)"
+	exit 0
+fi
+
+echo "removing ad server deployment again..."
```

Comment:
Very nitpicky, but the word "again" here strikes me as unnecessary.

Reply:
right, I thought I had changed that...

Reply:
done